packages/amplify-graphql-index-transformer/src/resolvers.ts

```diff
@@ -891,9 +891,10 @@ export const getDeltaSyncTableTtl = (resourceOverrides: $TSAny, resource: Transf
 }
 
 export const getResourceOverrides = (transformers: TransformerPluginProvider[], stackManager?: StackManagerProvider | null): $TSAny => {
-  try {
+  if (stateManager.currentMetaFileExists(undefined)) {
```

Comment:
this was getting swallowed by previous try-catch block.

https://app.circleci.com/pipelines/github/aws-amplify/amplify-category-api/2216/workflows/f85fb86b-64ca-4939-aa83-b9471c0b9e5d/jobs/51611

```
ProjectNotInitializedError: No Amplify backend project files detected within this folder.

      892 |
      893 | export const getResourceOverrides = (transformers: TransformerPluginProvider[], stackManager?: StackManagerProvider | null): $TSAny => {
    > 894 |   const meta = stateManager.getCurrentMeta(undefined, { throwIfNotExist: false });
          |                             ^
      895 |   const gqlApiName = _.entries(meta?.api).find(([, value]) => (value as { service: string }).service === 'AppSync')?.[0];
      896 |   if (gqlApiName && stackManager) {
      897 |     const backendDir = pathManager.getBackendDirPath();

      at projectNotInitializedError (../../node_modules/amplify-cli-core/src/errors/project-not-initialized-error.ts:6:63)
      at PathManager.constructPath (../../node_modules/amplify-cli-core/src/state-manager/pathManager.ts:280:37)
      at PathManager.getCurrentAmplifyMetaFilePath (../../node_modules/amplify-cli-core/src/state-manager/pathManager.ts:201:74)
      at StateManager.getCurrentMeta (../../node_modules/amplify-cli-core/src/state-manager/stateManager.ts:57:34)
      at getResourceOverrides (src/resolvers.ts:894:29)
      at IndexTransformer.after (src/graphql-index-transformer.ts:72:53)
      at GraphQLTransform.transform (../amplify-graphql-transformer-core/src/transformation/transform.ts:295:21)
      at Object.<anonymous> (src/__tests__/amplify-graphql-index-transformer.test.ts:1153:27)
```